config/webpack/webpack.config.js

```diff
@@ -29,6 +29,8 @@ const {
   sourceMapsProd,
   supportedBrowsers,
   displayNamesProd,
+  cspEnabled,
+  cspExtraHosts,
```

Comment:
Similar to the above question, is it worth clarifying this is for scripts, e.g. `cspExtraScriptSrcHosts` ? Bit of a mouthful though